<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>789</width>
      <height>160</height>
     </rect>
    </property>
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QListWidget" name="imageList">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>591</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loadImagesButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dateSortButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Order By Date</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nameSortButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Order By Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>779</width>
      <height>160</height>
     </rect>
    </property>
    <property name="title">
     <string>Class</string>
    </property>
    <widget class="QPushButton" name="classLoad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Class</string>
     </property>
    </widget>
    <widget class="QListWidget" name="classList">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>581</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="classEnter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Add Class Name Then Press &quot;Add&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="classRemove">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Class</string>
     </property>
    </widget>
    <widget class="QPushButton" name="classAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Class</string>
     </property>
    </widget>
    <widget class="QPushButton" name="classSave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Classes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="classAsc">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort A-Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>0</y>
      <width>331</width>
      <height>160</height>
     </rect>
    </property>
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QPushButton" name="saveYOLOButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save YOLO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveVOCButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save VOC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveCOCOButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save COCO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restoreButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="editVOCName">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>VOC Folder</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="imageDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>1901</width>
      <height>841</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>1900</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BBoxes:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadBboxButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Load File</string>
     </property>
    </widget>
    <widget class="QComboBox" name="shapeCombo">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>291</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Pick a shape...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square/Rectangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Trapezium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polygon</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Shapes:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="editShapeButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Shape</string>
     </property>
    </widget>
    <widget class="QPushButton" name="helpButton">
     <property name="geometry">
      <rect>
       <x>1740</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="copyButton">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pasteButton">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Paste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cropButton">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Crop/Save</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>imageDisplay</zorder>
   <zorder>groupBox_4</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
